backend/src/schema/resolvers/images/images.js

```diff
@@ -0,0 +1,121 @@
+import path from 'path'
+import { v4 as uuid } from 'uuid'
+import slug from 'slug'
+import { existsSync, unlinkSync, createWriteStream } from 'fs'
+import { getDriver } from '../../../db/neo4j'
+import { UserInputError } from 'apollo-server'
+
+// const widths = [34, 160, 320, 640, 1024]
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-25T14:37:05Z" title="Tuesday, February 25th 2020, 3:37:05 pm +01:00">Feb 25, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,120 @@
+import path from 'path'
+import uuid from 'uuid/v4'
+import slug from 'slug'
+import { existsSync, unlinkSync, createWriteStream } from 'fs'
+import { getDriver } from '../../../db/neo4j'
+import { UserInputError } from 'apollo-server'
+
+// const widths = [34, 160, 320, 640, 1024]
```


it's not clear at all whether you favor multiple options uploaded to spaces, or resizing on the fly :stuck_out_tongue: